### ISSUE TITLE
docs: link from README to docs?

### ISSUE THREAD

**Issue by jorgeorpinel** [iterative/dvc-task#75]
There's a GHP docs site in https://docs.iterative.ai/dvc-task/ with an auto-generated API ref. but I don't think there's a way for users to find these docs.

Maybe add a basic index page and link from the README?

**Comment by pmrowla** [comment]
The API ref isn't empty. I'm assuming it's because of the mobile stylesheet, but the API reference should be viewable if you use the nav bar

![Screenshot 2022-07-08 013011](https://user-images.githubusercontent.com/651988/177824379-8eab7ddd-1276-4ace-9146-53e8582dcbe5.png)

**Comment by jorgeorpinel** [comment]
I see. You need a pretty large screen for the nav to display all the objects. Otherwise they're hidden in the menu.

In any case does the site have any users? The main point of this issue is to link it from the README or from somewhere so people can find it.

**Comment by pmrowla** [comment]
> In any case does the site have any users?

No. The autogenerated docs still are still experimental, questions regarding the docs UI (we are basically just using default templates afaik) should go in the template repo: https://github.com/iterative/py-template/issues/19

There's still ongoing discussion as to whether or not we should be using the autogeneration tool that is currently configured (mkdocs) or something else (sphinx)
